compliance_checker/cf/util.py

```diff
@@ -261,7 +262,7 @@ def _get(self, entrynode, attrname, required=False):
     def __init__(self, cached_location=None):
         if cached_location:
             try:
-                with open(cached_location, 'r') as fp:
+                with io.open(cached_location, 'r', encoding='utf-8') as fp:
```

Comment:
There's a long, and probably never ending debate about exception handling among software engineers and computer scientists. On the one hand, and I'm in this boat, I like the philosophy, "fail early and often, so we can fix it". The other boat is larger, and has far more people in it, and they all say "don't punish the users, hide the failures, fail gracefully and if you can, try to log it." 

I would prefer not catching all exceptions in all but very rare cases that are absolutely clear and documented.